README.md

```diff
@@ -1,110 +1,7 @@
-# Overview
+[![Netlify Status](https://api.netlify.com/api/v1/badges/a76820ec-1092-49fb-8538-a82900b711b0/deploy-status)](https://app.netlify.com/sites/aswf-tac/deploys)
 
-The role of the Technical Advisory Committee (TAC) of the Academy Software Foundation (ASWF) is to facilitate communication and collaboration among the Technical Projects. The TAC will be responsible for:
+# Academy Software Foundation Techincal Advisory Council (TAC)
```

Comment:
Typo: "Technical", not "Techincal"

Reply:
The bane of my spelling challenges - thank you :-)